docs/leo-rover/integrations/robotic-arms/trossen-widowx-250.mdx

```diff
@@ -188,16 +206,14 @@ point to your result space. Open the file in `nano`:
 sudo nano /etc/ros/setup.bash
 ```
 
-Comment out the first line by adding `#` sign and add the source command for
-your workspace. The first 2 lines should look essentially like this:
+Add below line to this file:
```

Comment:
Why did you change this line?

Reply:
Must have omitted this while copying, I'll revert it so it's more similar to original tutorial